src/clj/kefirnadar/application/queries.clj

```diff
@@ -3,24 +3,22 @@
     [datomic.client.api :as d]))
 
 
-(defn user                                                  ;; currently, not used, maybe in future
+(defn ad                                                  ;; currently, not used, maybe in future
   "Find a user based on its id."
   [db user-id]
   (d/pull db [:*] user-id))
 
-
-(defn users
+(defn ads
   ([db]
-   (users db nil))
+   (ads db nil))
   ([db cond]
    (println "COND: " cond)
-   (let [q (cond->
-             '{:find  [(pull ?eid [*])]
-               :where [[?eid :ad/created]]}
-             (map? cond) (update :where into (map #(into ['?eid] %)) cond))]
+   (let [condition-vector (mapv (fn [cond-pair]
```

Comment:
ovo je jako teško razumeti šta radi. 90% sam siguran da može jednostavnije da se uradi. Pogotovo je nejasno zato što se ne vidi ni koji oblik parametara prosleđuješ pri pozivu ove funkcije. Šta ova funkcija radi? Zašto moramo dinamički da pravimo upit?